### ISSUE TITLE
INFO:statsite.influxdb:204, No Content seems like an error, but it just means InfluxDB was successful

### ISSUE THREAD

**Issue by RichardHightower** [armon/statsite#152]
INFO:statsite.influxdb:204, No Content sounds like an error and it prints out a lot. 

It is the No Content part that is confusing. 
It seems like something went wrong even though according to the InfluxDB docs that means success.
https://influxdb.com/docs/v0.9/guides/writing_data.html

```
HTTP response summary
2xx: If it’s HTTP 204 No Content, success! I
```

Maybe the INFO log can just check for 204, and say success. :)

**Comment by kuba--** [comment]
This is just a log info with response code and reason. In my opinion INFO log level means it's not an error. On the other hand I do not know why InfluxDB chose that convention, but it sounds confusing, because "200 OK" means that "InfluxDB understood the request but couldn’t complete it".

**Comment by kuba--** [comment]
Anyway, for 0.9 I may add if-else logging (if you think it will be useful):

2xx: If it’s HTTP 204 No Content, success! If it’s HTTP 200 OK, InfluxDB understood the request but couldn’t complete it. The body of the response will contain additional error information.
4xx: InfluxDB could not understand the request.
5xx: The system is overloaded or significantly impaired.

**Comment by RichardHightower** [comment]
Agreed that 200 for Ok is off. But that is an InfluxDB issue. IMO it should return a 500 for I could not complete the request. I guess (not don't know) that not being able to complete the request might not be an exception in all cases. No Content IMO sounds like something went wrong when in fact, that is the only right answer. It was confusing. And I agree the problem is really with InfluxDB but no use propagating in statsite.

**Comment by armon** [comment]
This is fixed by #153

**Comment by RichardHightower** [comment]
thanks
